slides/src/content/es6.md

```diff
@@ -4,16 +4,45 @@
 
 ## Classes
 
+- Makes ES5 prototypical inheritance model function more like a traditional class-based language
+- Supports inheritance using the keyword `extend`
+- Supports constructors using the keyword `constructor`
+
 ```js
 class Hamburger {
-  constructor() {} // constructor
-  listToppings() {} // method
+  constructor() { }   // constructor 
+  listToppings() { }  // method
 }
+```
+
+- Methods of the class can be accessed using a class object. 
+
+```js
 let burger = new Hamburger();
 burger.listToppings();
 ```
 
-- Makes ES5 prototypical inheritance model function more like a traditional class-based language
+- getter and setter are supported using `get` / `set`
+
+```js
+class Hamburger {
+  ...
+  get toppings() {
```

Comment:
Maybe we can do some modification to demonstrate the value of a getter vs public property.  Perhaps 
```
get toppings() {
    return 'toppings are ' + this.toppings;
}
```

And in the usage example put comment to say this will return 'toppings are [toppings]'